.github/workflows/setup-test-workspace/action.yml

```diff
@@ -1,9 +1,8 @@
 name: Setup test workspace
 inputs:
   ref-slice-ref:
-    description: 'Reference of the ref_slice to checkout'
+    description: 'Git reference (branch name, tag or commit hash) of the ref_slice test fork to checkout'
     required: false
```

Comment:
Should this be `required: true`? Otherwise, what's the behavior if this input value isn't supplied?

Any time we make something optional, we're adding more modes of operation -- which means more things to test, and more things that can fail. Are we losing anything if we make this value required?

Reply:
My aim was to make the default behavior the same as for the `checkout` action, but you're right - since we don't use it, it's better to make the input required then wonder what the default behavior exactly is and try documenting it here.